setup.cfg

```diff
@@ -72,8 +73,9 @@ doc =
     myst-nb
     numpydoc
 dev =
+    pre-commit
```

Comment:
Wondering if we should move `pre-commit` to test dependencies (both here and in `dev-environment.yml`). 
I've had some feedback that is unpractical to have to add it separately when doing the `conda` dev installation `mamba env create -f dev-environment.yml`. Would be more practical?
I don't know if there's a recommended practice for this.